apps/examples/testcase/le_tc/kernel/kernel_tc_main.c

```diff
@@ -194,6 +194,44 @@ int tc_kernel_main(int argc, char *argv[])
 	umm_heap_main();
 #endif
 
+#ifdef CONFIG_ITC_KERNEL_ENVIRON
+	itc_environ_main();
+#endif
+
+#ifdef CONFIG_ITC_KERNEL_LIBC_PTHREAD
+	itc_libc_pthread_main();
+#endif
+
+#ifdef CONFIG_ITC_KERNEL_LIBC_SEMAPHORE
+	itc_libc_semaphore_main();
+#endif
+
+#ifdef CONFIG_ITC_KERNEL_SEMAPHORE
+#if (!defined CONFIG_DEBUG) || (!defined CONFIG_SEM_PREALLOCHOLDERS) || (!defined CONFIG_PRIORITY_INHERITANCE)
+//#error CONFIG_DEBUG, CONFIG_SEM_PHDEBUG, CONFIG_SEM_PREALLOCHOLDERS and CONFIG_PRIORITY_INHERITANCE are needed for testing SEMAPHORE ITC
```

Comment:
if not needed, please remove this line.

Reply:
removed unused code as per your comments